test/supply-test.ts

```diff
@@ -436,10 +554,11 @@ describe('supplyTo', function () {
     await EVIL.setAttack(attack);
 
     await comet.connect(alice).allow(EVIL.address, true);
-
+    await wait(EVIL.connect(alice).approve(comet.address, 75e6));
+    await EVIL.allocateTo(alice.address, 75e6);
     await expect(
       comet.connect(alice).supplyTo(bob.address, EVIL.address, 75e6)
-    ).to.be.revertedWith("custom error 'SupplyCapExceeded()'");
+    ).to.be.revertedWith("custom error 'TransferInFailed()'");
```

Comment:
Now the accounting in comet won't be inflated by re-entrancy supply reaching SupplyCapExceeded(). 
It will be blocked and throw TransferInFailed() error.